src/code/z_actor.c

```diff
@@ -2533,40 +2536,40 @@ void func_80031C3C(ActorContext* actorCtx, GlobalContext* globalCtx) {
 }
 
 /**
- * Adds a given actor instance at the front of the actor list of the specified type.
- * Also sets the actor instance as being of that type.
+ * Adds a given actor instance at the front of the actor list of the specified category.
+ * Also sets the actor instance as being of that category.
  */
-void Actor_AddToTypeList(ActorContext* actorCtx, Actor* actorToAdd, u8 actorType) {
+void Actor_AddToCategoryList(ActorContext* actorCtx, Actor* actorToAdd, u8 actorCategory) {
     Actor* prevFirstActor;
```

Comment:
Also I suggest renaming the different variables in these functions to use "head" instead of "first".
For instance:
```suggestion
    Actor* prevHead;
```